URA-695/cell_line_analysis.py

```
@@ -0,0 +1,258 @@
+    title_file = "plots/" + title + "_boxplots_" + str(query_col) + "_" + str(query_col2) + "_" + str(truth_col) + ".jpg"
+
+    # Create a figure and axis
+    fig, axes = plt.subplots(nrows=1, ncols=3, figsize=(15, 5))
+
+    for ax, group_data, title in zip(axes, data, [query_col, query_col2, truth_col]):
+        sns.boxplot(data=group_data, ax=ax, color='lightblue')
+        sns.stripplot(data=group_data, ax=ax, color='black', alpha=0.5, jitter=True)
+        ax.set_title(title)
+        ax.set_ylabel('Allele Frequency')
+        ax.set_xticks([]) 
+        ax.text(0.1, 0.95, 'n: {}'.format(len(group_data)),
```

Ruff (0.8.0)

Use f-string instead of `format` call

Convert to f-string

(UP032)